packages/ui/package.json

```
@@ -41,6 +41,7 @@
 		"@radix-ui/react-alert-dialog": "1.0.5",
 		"@radix-ui/react-aspect-ratio": "1.0.3",
 		"@radix-ui/react-avatar": "1.0.4",
+		"@radix-ui/react-checkbox": "^1.0.4",
```

```suggestion
		"@radix-ui/react-checkbox": "1.0.4",
```

em pin version cho anh rồi generate lại lock file nhé